src/dll/Hooks/ValidateScripts.cpp

```diff
@@ -37,10 +36,11 @@ bool _ScriptValidator_Validate(uint64_t self, uint64_t a1, RED4ext::ScriptReport
         }
     }
 
-    auto message = WritePopupMessage(errors);
+    const auto& incompatiblePlugins = App::Get()->GetPluginSystem()->GetIncompatiblePlugins();
+    const auto message = WritePopupMessage(validationErrors, incompatiblePlugins);
     if (!message.empty())
     {
-        MessageBoxA(0, message.c_str(), "Script Validation Error", MB_OK | MB_ICONERROR);
+        MessageBoxW(0, message.c_str(), L"Script Validation Error", MB_OK | MB_ICONERROR);
```

Comment:
I suggest to use [SHOW_MESSAGE_BOX_AND_EXIT_FILE_LINE](https://github.com/WopsS/RED4ext/blob/master/src/dll/Utils.hpp#L93) instead of this. That one is used everywhere in RED4ext from what I remember.